includes/wikia/models/PromoImage.class.php

```diff
@@ -128,34 +102,16 @@ public function corporateFileByLang($lang){
 		return GlobalFile::newFromText($this->getPathname(), $wiki_id);
 	}
 
-	public function isFileChanged(){
-		return !empty($this->fileChanged);
-	}
-
-	public function processUploadedFile($srcFileName) {
-		if ($this->isValid()){ // do not upload invalid filenames
-			$this->fileChanged = true;
-			$dst_file_title = Title::newFromText($this->getPathname(), NS_FILE);
-
-			$temp_file = RepoGroup::singleton()->getLocalRepo()->getUploadStash()->getFile($srcFileName);
-			$file = new LocalFile($dst_file_title, RepoGroup::singleton()->getLocalRepo());
-
-			$file->upload($temp_file->getPath(), '', '');
-			$temp_file->remove();
-		}
-		return $this;
-	}
-
-	static protected function inferType( $fileName, &$dbName = null ) {
+	static private function inferType( $fileName, &$dbName = null ) {
 		$pattern = "/^(".self::__MAIN_IMAGE_BASE_NAME.")?(".self::__ADDITIONAL_IMAGES_BASE_NAME."-(\d)?)?,?([^.]{1,})?\.?(.*)$/i";
 		$type = self::INVALID;
 
 		if (preg_match($pattern, $fileName, $matches)){
 			if (!empty($matches[1])) {
 				$type = self::MAIN;
-			} elseif (!empty($matches[2]) and !empty($matches[3])) { // matches additional images and has a number designation
+			} elseif (!empty($matches[2]) && !empty($matches[3])) { // matches additional images and has a number designation
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)